docs/dev/api_usage_examples/get_latest_import.py

```diff
@@ -0,0 +1,41 @@
+'''
+get_latest_import.py
+'''
+
+from common import AUTH_HEADER, USERNAME, construct_url
```

Comment:
Since these are examples, I'd prefer to keep these explicit. So that people can just look at the `submit_listen` code and understand it, without having to go into `common.py` to understand what the `AUTH_HEADER` is exactly.

Reply:
`common.py` is fairly short, and it's given and the top of `api-usage.rst`. Should I just copy the required things that are in `common.py` right into the start of every file? That seems excessive to me, and it would only increase the size of the examples.

Reply:
I think if we create examples that are easy to read individually without having to read the entire document, it would be better. That is why I suggested not using imports because then I have to go up to see what `AUTH_TOKEN` means, which is extra baggage if I'm a random guy who just wants to see how to submit listens and move on.